src/views/pages/items.jsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import ListenerComponent from '../../lib/listener-component.jsx';
+import ItemStore from '../../store/item-store.js';
+import { viewModel } from '../../lib/api-client.js'
+import { getModelsForNewItem } from '../../lib/api-client';
+import Item from '../components/item.jsx';
+export default class Models extends ListenerComponent {
+
+    constructor() {
+        super();
+
+    }
+    getStores() {
+        return [
+            ItemStore
+        ];
+    }
+    getState() {
+        return {
+            items: ItemStore.getAllItems()
+        };
+    }
+    newItem(e) {
```

Comment:
Instead of `onClick={this.newItem.bind(this)}`, use `onClick={getModelsForNewItem}`.